config/samples/infrastructure_v1alpha1_ionoscloudloadbalancer.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha1
+kind: IonosCloudLoadBalancer
+metadata:
+  labels:
+    app.kubernetes.io/name: cluster-api-provider-ionoscloud
+    app.kubernetes.io/managed-by: kustomize
+  name: ionoscloudloadbalancer-sample
+spec:
+  # TODO(user): Add fields here
```

Comment:
please provide a working sample spec

Reply:
We haven't done that before. Also samples will be in the cluster template. Why do you specifically want this here now?

Reply:
i wasn't aware that this wasn't done before.